mega-zep-frontend/src/main/angular/frontend/src/app/modules/shared/interceptors/global-http-interceptor.service.ts

```diff
@@ -1,24 +1,17 @@
-import { Injectable } from '@angular/core';
-import { HttpErrorResponse, HttpEvent, HttpHandler, HttpInterceptor, HttpRequest } from '@angular/common/http';
-import { Observable, of, throwError } from 'rxjs';
-import { catchError, tap } from 'rxjs/operators';
-import { ErrorHandlerService } from '../services/error/error-handler.service';
-import { UserService } from '../services/user/user.service';
-import { ConfigService } from '../services/config/config.service';
-import { LoaderService } from '../services/loader/loader.service';
+import {Injectable} from '@angular/core';
+import {HttpErrorResponse, HttpEvent, HttpHandler, HttpInterceptor, HttpRequest} from '@angular/common/http';
+import {Observable, of} from 'rxjs';
+import {catchError, tap} from 'rxjs/operators';
+import {ErrorHandlerService} from '../services/error/error-handler.service';
+import {UserService} from '../services/user/user.service';
+import {ConfigService} from '../services/config/config.service';
+import {LoaderService} from '../services/loader/loader.service';
 
 @Injectable({
   providedIn: 'root'
 })
 export class GlobalHttpInterceptorService implements HttpInterceptor {
 
-  private readonly HTTP_STATUS_BAD_REQUEST: number = 400;
-  private readonly HTTP_STATUS_UNAUTHORIZED: number = 401;
-  private readonly HTTP_STATUS_FORBIDDEN: number = 403;
-  private readonly HTTP_STATUS_NOT_FOUND: number = 404;
-  private readonly HTTP_STATUS_METHOD_NOT_ALLOWED: number = 405;
-  private readonly HTTP_STATUS_REQUEST_TIMEOUT: number = 408;
-
   constructor(
     private errorHandler: ErrorHandlerService,
```

Comment:
we should lookup documentation on this code, not necessary a problem with this pull request ... is the direct call of handleError from application code valid?